libtransmission/torrent.h

```diff
@@ -898,6 +913,8 @@ struct tr_torrent final : public tr_completion::torrent_view
 
     bool finishedSeedingByIdle = false;
 
+    bool sequentialDownload = false;
```

Comment:
I _think_ this field can be made private?

Please try moving this field to the end of the class next to `needs_completeness_check_` and use the same private field naming scheme, `bool sequential_download_ = false;`